bindings/java/pom.xml.in

```diff
@@ -188,7 +188,7 @@
   <build>
     <sourceDirectory>${jucx.src.dir}/src/main/java</sourceDirectory>
     <testSourceDirectory>${jucx.src.dir}/src/test/java</testSourceDirectory>
-    <directory>${native.dir}/build-java</directory>
+    <directory>${native.dir}</directory>
```

Comment:
This is the directory where maven puts compiled classes/jars for this build. Why to change it?

Reply:
a leftover, will remove

Reply:
@petro-rudenko fixed